run-checks

```diff
@@ -158,15 +158,18 @@ if [ "$STATIC" = 1 ]; then
     if [ -z "${SKIP_GOFMT:-}" ]; then
         echo Checking formatting
         fmt=""
-        for dir in $(go list -f '{{.Dir}}' ./... | grep -v '/vendor/' ); do
-            s="$(${GOFMT:-gofmt} -s -l -d "$dir" | grep -v /vendor/ || true)"
+        for dir in $(go list -f '{{.Dir}}' ./... | grep -v '/vendor/' | grep -E 'snapd/[A-Za-z0-9_]+$' ); do
```

Comment:
We could remove the "grep -v /vendor/" now because the other grep already excludes it. Might be simpler to read with keeping the exclude though.

Reply:
I have no preference here, happy to adjust.